src/common/ios/plistBuddy.ts

```diff
@@ -16,7 +16,7 @@ export class PlistBuddy {
 
     constructor({
         nodeChildProcess = new Node.ChildProcess(),
-        xcodeproj = new Xcodeproj()
+        xcodeproj = new Xcodeproj(),

```

Comment:
We have a tslint that validate that.
It makes it easier to avoid conflicts if you add another element to the array/object literal